src/ProgressBar.js

```diff
@@ -1,33 +1,37 @@
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
 import cx from 'classnames';
 
-class ProgressBar extends React.Component {
-  render() {
-    let classes, style;
-    if(typeof this.props.progress === 'undefined') {
-      classes = {indeterminate: true};
-    } else {
-      classes = {determinate: true};
+class ProgressBar extends Component {
+  render () {
+    const { progress, className } = this.props;
+
+    let style;
+    const classes = {
+      indeterminate: !progress,
+      determinate: !!progress
+    };
+    if (progress) {
       style = {
-        width: this.props.progress + '%',
+        width: progress + '%'
       };
     }
 
     return (
-      <div className={cx('progress', this.props.className)}>
-        <div className={cx(classes)} style={style}></div>
+      <div className={cx('progress lighten-5', className)}>
```

Comment:
in the progress bar, the progress is marker in X color and the remaining progress is lighten-5 of X